src/script/auth/page/Login.tsx

```diff
@@ -238,6 +239,11 @@ class Login extends React.Component<CombinedProps, State> {
           ? this.props.doLoginAndJoin(login, this.state.conversationKey, this.state.conversationCode)
           : this.props.doLogin(login);
       })
+      .then(() => {
+        const secretKey = new Uint32Array(64);
+        self.crypto.getRandomValues(secretKey);
+        return save(secretKey);
```

Comment:
This is mostly demo code (since we are not sure yet how this is going to be consumed)